### PR TITLE
[Documentation] Behaviour Feedback widgets description

### LINKED ISSUE #6630: [bvl_feedback] Behaviour Feedback module does not describe or test widgets
The bvl_feedback module registers widgets to the dashboard, but the README does not describe them and the test plan does not test them.

Related to #6467

### PR DESCRIPTION
**Pull request by laemtl**
Tentative to update the documentation of the bvl_feedback module:
- creation of a basic README
- update of the Test Plan to mention the Behavioral Feedback Notifications widget.

* Resolves #6630

### CODE REVIEW

File: modules/bvl_feedback/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Behavioural Feedback
+
+
+## Purpose
+
+The Feedback module allows users to flag and comment on any database field particularly for behavioural data entry.
```
Comment (driusan) [inline modules/bvl_feedback/README.md:6]:
```suggestion
The Feedback module allows users to flag and comment on field behavioural database fields. 
```

File: modules/bvl_feedback/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Behavioural Feedback
+
+
+## Purpose
+
+The Feedback module allows users to flag and comment on any database field particularly for behavioural data entry. 
+Two default feedback types exist: input and scoring errors. Additional feedback types can be added to feedback_bvl_type table.
```
Comment (driusan) [inline modules/bvl_feedback/README.md:7]: This should be under a `## Configurations` section, not the purpose.

File: modules/bvl_feedback/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+## Behavioral Feedback Notifications
+
+The Dashboard Widget tracks issues and corrections to behavioural/clinical data points. Notification are clickable and redirect to the instrument with new changes or feedback.
```
Comment (driusan) [inline modules/bvl_feedback/README.md:11]: This should be in an interactions section describing how it registers widgets on the dashboard, the description of being clickable is unnecessary for the spec. It's an incidental part of the implementation, not a core part of the design.

File: modules/bvl_feedback/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Behavioural Feedback
+
+
+## Configurations
+
+The Feedback module allows users to flag and comment on  behavioural database fields.
```
Comment (driusan) [inline modules/bvl_feedback/README.md:6]: The first paragraph is still the purpose, it's just the second that I was saying is a configuration

**Comment by christinerogers** [comment]
@driusan pls re-review

**Pull request merged**